docs/configuration/configuration.md

```diff
@@ -1719,6 +1719,10 @@ url: <string>
 # Timeout for requests to the remote write endpoint.
 [ remote_timeout: <duration> | default = 30s ]
 
+# Custom HTTP headers to be sent along with each remote write request.
```

Comment:
You miss a warning here that not all headers can be overwritten.

Reply:
Just added thanks!